touchin-cell/Cell/cellViewModel.swift.liquid

```diff
@@ -0,0 +1,6 @@
+import RxSwift
+import RxCocoa
+
+final class {{ module_info.name }}ViewModel {
```

Comment:
final class without inheritance almost the same as struct. use struct by default?

Reply:
No. Struct got different purpose.